### PR TITLE
feat(@angular-devkit/build-angular): update webpack dependencies

### LINKED ISSUE #12774: Autoprefixer IE11 CSS Grid Support Removed
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [ ] new
- [x] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Versions

node: 11.0.0
npm: 6.4.1
ng: 7.0.1
os: Windows 10

### Repro steps

Build an app containing CSS grids with v7.0.0-7.0.3 of the Angular CLI and access it in IE11. Inspect the styles to see that no grid tags are prefixed with -ms.

### The log given by the failure

N/A

### Desired functionality

Previously, autoprefixer grid support was turned on by default. This functionality was removed in #11453 without a way to configure the functionality back on. I propose one or both of the following solutions: 

- Add a build configuration option to enable autoprefixer CSS grid support across the app.
- Update autoprefixer to version 9.2 or later, which includes support for [grid control comments](https://github.com/postcss/autoprefixer/releases/tag/9.2.0)

### Mention any other details that might be useful

I can PR one or both of these changes if needed but would like feedback on the proposed solutions.

### PR DESCRIPTION
**Pull request by alan-agius4**
Closes #12774

### CODE REVIEW

**Comment by gerbermichi** [comment]
Can I do something to speed up this pull request? We depend on the new autoprefixer, otherwise our css grid wont work in IE. :(

**Comment by filipesilva** [comment]
@gerbermichi at this point it's approved and green. It should be merged by the caretaker soon and released this week in `7.1.0-beta.1`.

File: packages/angular_devkit/build_angular/package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "webpack-dev-server": "3.1.10",
     "webpack-merge": "4.1.4",
     "webpack-sources": "1.3.0",
-    "webpack-subresource-integrity": "1.1.0-rc.6"
+    "webpack-subresource-integrity": "1.2.0"
```
Comment (clydin) [inline packages/angular_devkit/build_angular/package.json]: Leave this at the existing version.  1.2.0 has an unnecessary peer dependency on `html-webpack-plugin` which will cause a peer dependency warning for every CLI user.

Comment (alan-agius4) [reply packages/angular_devkit/build_angular/package.json]: @clydin updated.

**Pull request merged**